db/schema.sql

```diff
@@ -1234,6 +1234,13 @@ CREATE INDEX header_sync_transactions_header ON public.header_sync_transactions
 CREATE INDEX headers_block_number ON public.headers USING btree (block_number);
 
 
+--
+-- Name: headers_check_count; Type: INDEX; Schema: public; Owner: -
+--
+
+CREATE INDEX headers_check_count ON public.headers USING btree (check_count);
+
+
 --
```

Comment:
I think the `header_sync_logs_untransformed` index is missing from the schema changes

Reply:
Nice catch! Updated 👍